core/kernel/tee_ta_manager.c

```diff
@@ -261,6 +261,54 @@ static void tee_ta_unlink_session(struct tee_ta_session *s,
 	mutex_unlock(&tee_ta_mutex);
 }
 
+static void destroy_session(struct tee_ta_session *s,
+			    struct tee_ta_session_head *open_sessions)
+{
+	tee_ta_unlink_session(s, open_sessions);
+#if defined(CFG_TA_GPROF_SUPPORT)
+	free(s->sbuf);
+#endif
+	free(s);
+}
+
+static void destroy_ta_ctx_from_session(struct tee_ta_session *sess,
+				struct tee_ta_session_head *open_sessions)
+{
+	struct tee_ta_session *s;
```

Comment:
minor: default init to `NULL`, even if useless.
By convention in OP-TEE core implementation, all function local variables shall be initialized when defined.